### PR TITLE
Resubm

### PR DESCRIPTION
**Pull request by Farnazmdi**
Figure 6 updates. I also updated the Sankey figure a little bit. The one I had written was not general for both drugs.
image

### CODE REVIEW

File: lineage/Analyze.py [hunk 1]
```diff
@@ -151,6 +151,7 @@ def Results(tHMMobj, pred_states_by_lineage: list, LL: float) -> dict[str, Any]:
     results_dict["state_counter"] = np.bincount(ravel_switched_pred_states)
     results_dict["state_proportions"] = [100 * i / len(ravel_switched_pred_states) for i in results_dict["state_counter"]]
     results_dict["state_proportions_0"] = results_dict["state_proportions"][0]
+    results_dict["accuracy_before_switching"] = 100 * np.mean(ravel_true_states == pred_states_by_lineage)
```
Comment (aarmey) [inline lineage/Analyze.py:154]: Do you need this anymore?

File: lineage/Analyze.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ def Results(tHMMobj, pred_states_by_lineage: list, LL: float) -> dict[str, Any]:
     for ii, switcher in enumerate(switcher_map_holder):
         sw_states = [[switcher[st] for st in st_ass] for st_ass in pred_states_by_lineage]
         switcher_LL_holder[ii] = np.sum(tHMMobj.log_score(sw_states, pi=pi_arg, T=T_arg, E=E_arg))
-
+        results_dict[str("log_score"+str(ii))] = np.sum(tHMMobj.log_score(sw_states, pi=pi_arg, T=T_arg, E=E_arg))
```
Comment (aarmey) [inline lineage/Analyze.py:118]: Do you need this anymore?

File: lineage/figures/figure6.py [hunk 3]
```diff
@@ -40,7 +40,15 @@ def accuracy():
     """ A Helper function to create more random copies of a population. """
     # Creating a list of populations to analyze over
     list_of_Es = [[StateDistribution(E2[1].params[0], E2[1].params[1], E2[1].params[2], a, E2[1].params[4], E2[1].params[5]), E2[1]] for a in np.linspace(4.0, 20.0, num_data_points)]
-    list_of_populations = [[LineageTree.init_from_parameters(pi, T, E, max_desired_num_cells)] for E in list_of_Es]
+    list_of_populations = []
```
Comment (aarmey) [inline lineage/figures/figure6.py:43]: The abundance of the cell states should be changed by adjusting pi and T, not by filtering out lineages.

File: lineage/figures/figure6.py [hunk 4]
```diff
@@ -57,17 +65,17 @@ def accuracy():
     # replace x with 1-x if the accuracy is less than 50%
     balanced_score[balanced_score < 50.0] = 100.0 - balanced_score[balanced_score < 50.0]
 
-    wass, _, dict_out, _ = commonAnalyze(list_of_populations, 2, xtype="wass", list_of_fpi=[pi] * num_data_points, list_of_fT=[T] * num_data_points, parallel=True)
+    wass, _, dict_out, _ = commonAnalyze(list_of_populations, 2, xtype="wass", list_of_fpi=[pi] * 2*num_data_points, list_of_fT=[T] * 2*num_data_points, parallel=True)
```
Comment (aarmey) [inline lineage/figures/figure6.py:68]: Why do you have to do this?

**Review by aarmey — CHANGES_REQUESTED** [review]

**Comment by Farnazmdi** [comment]
I edited this branch so it only includes some edits to figure S17.

**Review by aarmey — APPROVED** [review]
Great!

**Pull request merged**